README.md

```diff
@@ -49,6 +49,11 @@ Enable or disable accepting ranged requests, defaults to true.
 Disabling this will not send `Accept-Ranges` and ignore the contents
 of the `Range` request header.
 
+##### acceptRanges
+
+Specify maximum chunk size, defaults to send the entire file size.
+This will be used when `acceptRanges` is true.
+
```

Comment:
This needs to be renamed to `maxChunkSize` and moved to be after `maxAge`, as the options are sorted alphabetically.

Reply:
Thanks, I also re-order the option in other files.